source/game_sa/CClock.cpp

```diff
@@ -1,38 +1,39 @@
 #include "StdInc.h"
 
-// 0x8CCF24
-unsigned char CClock::daysInMonth[12] = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
-// 0xB70144
-bool CClock::bClockHasBeenStored;
-unsigned short& CClock::ms_Stored_nGameClockSeconds = *reinterpret_cast<unsigned short*>(0xB70148);
-unsigned char& CClock::ms_Stored_nGameClockMinutes = *reinterpret_cast<unsigned char*>(0xB7014A);
-unsigned char& CClock::ms_Stored_nGameClockHours = *reinterpret_cast<unsigned char*>(0xB7014B);
-unsigned char& CClock::ms_Stored_nGameClockDays = *reinterpret_cast<unsigned char*>(0xB7014C);
-unsigned char& CClock::ms_Stored_nGameClockMonths = *reinterpret_cast<unsigned char*>(0xB7014D);
-unsigned char& CClock::CurrentDay = *reinterpret_cast<unsigned char*>(0xB7014E);
-unsigned short& CClock::ms_nGameClockSeconds = *reinterpret_cast<unsigned short*>(0xB70150);
-unsigned char& CClock::ms_nGameClockMinutes = *reinterpret_cast<unsigned char*>(0xB70152);
-unsigned char& CClock::ms_nGameClockHours = *reinterpret_cast<unsigned char*>(0xB70153);
-unsigned char& CClock::ms_nGameClockDays = *reinterpret_cast<unsigned char*>(0xB70154);
-unsigned char& CClock::ms_nGameClockMonth = *reinterpret_cast<unsigned char*>(0xB70155);
-unsigned int& CClock::ms_nLastClockTick = *reinterpret_cast<unsigned int*>(0xB70158);
-unsigned int& CClock::ms_nMillisecondsPerGameMinute = *reinterpret_cast<unsigned int*>(0xB7015C);
+uint8_t CClock::daysInMonth[12] = {31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31}; // 0x8CCF24
+
+bool CClock::bClockHasBeenStored;             // 0xB70144
+uint16_t CClock::ms_Stored_nGameClockSeconds; // 0xB70148
+uint8_t CClock::ms_Stored_nGameClockMinutes;  // 0xB7014A
+uint8_t CClock::ms_Stored_nGameClockHours;    // 0xB7014B
+uint8_t CClock::ms_Stored_nGameClockDays;     // 0xB7014C
+uint8_t CClock::ms_Stored_nGameClockMonths;   // 0xB7014D
```

Comment:
```suggestion
bool CClock::bClockHasBeenStored;             // 0xB70144
uint16_t CClock::ms_Stored_nGameClockSeconds; // 0xB70148
uint8_t CClock::ms_Stored_nGameClockMinutes;  // 0xB7014A
uint8_t CClock::ms_Stored_nGameClockHours;    // 0xB7014B
uint8_t CClock::ms_Stored_nGameClockDays;     // 0xB7014C
uint8_t CClock::ms_Stored_nGameClockMonths;   // 0xB7014D
```
It's not the right time to remove addresses. I sometimes unhook functions for debugging. It's very important to keep them for now.